scripts/catalog-tests/generate-tdr-snapshot-metadata.py

```diff
@@ -0,0 +1,127 @@
+#------------------------------------------------------------------------------
+# Generate TDR Snapshot Metadata
+#
+# This script adds snapshots to the catalog as the test user from DC-218,
+# pulling the metadata from the resources JSON file
+# 
+# Before Running:
+#   python3 -m venv .env
+#   source .env/bin/activate
+#   pip install --upgrade pip
+#   python3 -m pip install requests
+#   
+#   Authorization: Access the password in the vault using the appropriate command:
+#   vault login -method=github token=$(cat ~/.github-token)
+#   vault read secret/dsde/terra/kernel/dev/dev/catalog/tests/user
+#   vault read secret/dsde/terra/kernel/dev/dev/catalog/tests/userAdmin
+# Run: python3 generate-tdr-snapshot-metadata.py
+#------------------------------------------------------------------------------
+import json
+import os, subprocess, sys
+import requests
+
+urlRoot = os.environ.get('catalogServiceUrl') or 'http://localhost:8080'
+urlDatasets = f'{urlRoot}/api/v1/datasets'
+urlDataRepoList = "https://jade.datarepo-dev.broadinstitute.org/api/repository/v1/snapshots"
+user = os.environ.get('gcloudUser') or 'datacatalogadmin@test.firecloud.org'
```

Comment:
Does python map upper case to camel case environment vars for you? I would expect the environment variables to be `GCLOUD_USER`. Also the instructions should mention which env vars can/should be used.